packages/ssi-service/src/feedback-panel/feedback.js

```diff
@@ -0,0 +1,293 @@
+import html from 'html-template-tag';
+import { GraphQLClient } from 'graphql-request';
+import styles from './feedback.css';
+
+window.customElements.define('op-feedback', class extends HTMLElement {
+  constructor () {
+    super();
+
+    this.graphqlClient = new GraphQLClient('https://localhost:8080/graphql', {
+      headers: {
+        Authorization: 'Basic b25lX3BvcnRhbF9hcGk6PUw5dm5CK2RxNVoqcVo5cA=='
```

Comment:
Why are the credentials publicly pushed here? Please remove this asap!

Reply:
Removed.